README.md

```diff
@@ -66,6 +66,8 @@ collect.perf_schema.indexiowaits                       | 5.6           | Collect
 collect.perf_schema.tableiowaits                       | 5.6           | Collect metrics from performance_schema.table_io_waits_summary_by_table.
 collect.perf_schema.tablelocks                         | 5.6           | Collect metrics from performance_schema.table_lock_waits_summary_by_table.
 collect.slave_status                                   | 5.1           | Collect from SHOW SLAVE STATUS (Enabled by default)
+collect.pt-heartbeat                                   | 5.1           | Collect from pt-heartbeat
```

Comment:
I would prefer to call this `heartbeat`, as there could be other implementations.